cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1756,3 +1761,19 @@ def _create_course(test, course_key, course_data):
 def _get_course_id(course_data, key_class=SlashSeparatedCourseKey):
     """Returns the course ID (org/number/run)."""
     return key_class(course_data['org'], course_data['number'], course_data['run'])
+
+
+def _build_other_filter():
```

Comment:
The code coverage shows that there are no Python unit tests for this method:

https://jenkins.testeng.edx.org/job/edx-platform-report/8346/Diff_Coverage_Report/